src/game/GameManager.cpp

```diff
@@ -74,19 +111,27 @@ void GameManager::updateZombies(const float delta) {
 // Create marine add it to manager, returns marine id
 int32_t GameManager::createMarine() {
     const int32_t id = generateID();
-    marineManager[id] = Marine();
+    SDL_Rect marineRect = {initVal, initVal, defaultSize, defaultSize};
+    SDL_Rect moveRect = {initVal, initVal, defaultSize, defaultSize};
+    SDL_Rect projRect = {initVal, initVal, defaultSize, defaultSize};
+    SDL_Rect damRect = {initVal, initVal, defaultSize, defaultSize};
```

Comment:
Recommend moving `{initVal, initVal, defaultSize, defaultSize}` to a temp variable
This way it doesn't construct the same struct 4 times when it only needs to be once